src/ResourceManagement/Compute/ComputeManagement/Microsoft.Azure.Management.Compute.nuget.proj

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Management.Compute">
-      <PackageVersion>8.2.0</PackageVersion>
+      <PackageVersion>9.0.0</PackageVersion>
```

Comment:
Is this really a breaking changeIt looks like new OM, but I didn't see anybreaking changes to existing APIs, did I miss something?